ena_upload/ena_upload.py

```diff
@@ -362,28 +368,28 @@ def submit_data(file_paths, password, webin_id):
     print("\nConnecting to ftp.webin2.ebi.ac.uk....")
     try:
         ftps = MyFTP_TLS(timeout=10)
-        ftps.context.set_ciphers('DEFAULT@SECLEVEL=1')
+        ftps.context.set_ciphers('HIGH:!DH:!aNULL')
         ftps.connect(ftp_host, port=21)
         ftps.auth()
         ftps.login(webin_id, password)
         ftps.prot_p()
 
-    except IOError:
-        print(ftps.lastErrorText())
+    except IOError as ioe:
+        print(ioe)
```

Comment:
this print should be removed?

Reply:
since we moved to ftps we don't have `.lastErrorText()` anymore:

```
Exception has occurred: AttributeError
'MyFTP_TLS' object has no attribute 'lastErrorText'
  File "/home/bedro/My Documents/ena-upload-cli/ena_upload/ena_upload.py", line 366, in submit_data
    ftps.context.set_ciphers('DEFAULT@SECLEVEL=1')

During handling of the above exception, another exception occurred:

  File "/home/bedro/My Documents/ena-upload-cli/ena_upload/ena_upload.py", line 374, in submit_data
    print(ftps.lastErrorText())
  File "/home/bedro/My Documents/ena-upload-cli/ena_upload/ena_upload.py", line 820, in main
    submit_data(file_paths, password, webin_id)
  File "/home/bedro/My Documents/ena-upload-cli/ena_upload/ena_upload.py", line 901, in <module>
    main()
```

Reply:
I stumbled across this when I had an error on our cluster :)

Reply:
Ok, I see.